termwiz/src/escape/parser/mod.rs

```diff
@@ -173,6 +180,9 @@ impl<'a, F: FnMut(Action)> VTActor for Performer<'a, F> {
                 data: vec![],
             });
         } else {
+            if byte == b'p' && params == vec![1000] {
+                self.state.tmux_cc = true;
```

Comment:
I don't see anything that will exit this state, so as-written, this feels like it will "break" the pane when the mode is exited :-/

Reply:
I think in iTerm2 you need to manually break the state. In Wezterm I guess "press q"? I'll see where to add this.